app/view/fileupload/FileGrid.js

```diff
@@ -0,0 +1,92 @@
+Ext.define('CpsiMapview.view.fileupload.FileGrid',
+    {
+        extend: 'Ext.grid.Panel',
+        xtype: 'cmv_filegrid',
+        requires: [
+            'CpsiMapview.view.grid.ItemDeleter',
+            'Ext.util.Format',
+            //'CpsiMapview.view.fileupload.FileViewModel',
+            'CpsiMapview.view.fileupload.FileGridController'
+        ],
+        loadMask: true,
+        border: true,
+        anchor: '100% 100%',
+        multiSelect: true,
+        emptyText: 'No files to display',
+        reserveScrollOffset: true,
+        sortableColumns: false,
+        allowAddingAttachments: true,
+        //viewModel: 'cmv_fileviewmodel',
+        controller: 'cmv_filegridcontroller',
+        listeners: {
+            itemdblclick: 'onDownloadFileClick'
+        },
+        bind: {
+            store: '{files}',
+            serviceUrl: '{serviceUrl}'
+        },
+        setServiceUrl: function(value) {
```

Comment:
setServiceUrl and getServiceUrl can be replaced with:

```
config: {
    serviceUrl: ''
}
```

This will automatically create getters and setters - see https://docs.sencha.com/extjs/6.7.0/classic/Ext.Class.html#cfg-config